Modules/_ssl.c

```diff
@@ -2073,6 +2077,111 @@ cipher_to_dict(const SSL_CIPHER *cipher)
        );
 }
 
+/*[clinic input]
```

Comment:
This needs to be guarded by an ifdef because OpenSSL may be built without ECH support: https://github.com/openssl/openssl/compare/master...feature/ech#diff-6defafa2caa65304173956e3fbc26b6947c50bc769c08c3da32c1032fa6519d4R473.